prestodb/client.py

```diff
@@ -273,12 +271,12 @@ def process(self, http_response):
         if 'error' in response:
             raise self._process_error(response['error'])
 
-        if 'X-Presto-Clear-Session' in http_response.headers:
-            propname = response.headers['X-Presto-Clear-Session']
+        if constants.HEADER_CLEAR_SESSION in http_response.headers:
+            propname = response.headers[constants.HEADER_CLEAR_SESSION]
             self._session_properties.pop(propname, None)
```

Comment:
I am adding a `ClientSession` class and referencing it as `PrestoRequest._client_session`. `PrestoRequest.headers` is then a Python property attribute built from the latest state of the `ClientSession` instance.